package.json

```diff
@@ -48,8 +48,9 @@
     "@patternfly/react-table": "4.8.6",
     "@patternfly/react-tokens": "4.5.2",
     "@react-pdf/renderer": "1.6.9",
-    "@redhat-cloud-services/frontend-components": "2.1.3",
-    "@redhat-cloud-services/frontend-components-inventory-insights": "2.2.0",
+    "@redhat-cloud-services/frontend-components": "^2.3.1",
```

Comment:
gotta kill da carrot, we lock down on specific versons

Reply:
🔐